Makefile

```diff
@@ -372,7 +372,11 @@ clean: ## Cleanup local builds.
 	rm -rf $(BIN_DIR) $(PKG_CONFIG) $(TEST_RESULTS_DIR)
 
 $(PKG_CONFIG):
+ifeq ($(OVERRIDE_PKG_CONFIG), true)
+	@echo "Overriding pkg-config script. Use an alternate method to generate .pkg_config"
+else
```

Comment:
do we need to set the `ENABLED_GIT_COMMAND` for other targets that might come after this one?

Reply:
I'm not 100% following: are you suggesting we want to add

```bash
ENABLE_GIT_COMMAND=$(ENABLE_GIT_COMMAND)
```
to the override part of the target?